test/integration/query-interface/changeColumn.test.js

```diff
@@ -235,56 +235,7 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
     }
 
     if (dialect === 'sqlite') {
-      it('should not remove unique constraints when adding or modifying columns', async function() {
-        await this.queryInterface.createTable({
-          tableName: 'Foos'
-        }, {
-          id: {
-            allowNull: false,
-            autoIncrement: true,
-            primaryKey: true,
-            type: DataTypes.INTEGER
-          },
-          name: {
-            allowNull: false,
-            unique: true,
-            type: DataTypes.STRING
-          },
-          email: {
-            allowNull: false,
-            unique: true,
-            type: DataTypes.STRING
-          }
-        });
-
-        await this.queryInterface.addColumn('Foos', 'phone', {
```

Comment:
so before adding a column everything was previously just fine. but after the column, it would have lost the unique index somehow?

Reply:
Previously this test ensured single-column unique indexes persisted with a table modification. This PR rolls back the changes that make this test work.